### PR TITLE
bip: Add proposal's README as index page

### PR DESCRIPTION
**Pull request by abeaumont**

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -28,3 +28,9 @@ to the repository of the relevant component at the [bblfsh organization](https:/
 
 If you want to request a new feature, feel free to open it as an issue at the
 relevant GitHub repository.
+
+For deeper changes we use the [Babelfish Improvement Proposals](proposals/README.md).
+If the feature you want to request is quite complex
```
Comment (juanjux) [inline community.md]: What about: ...complex or could change some interfaces, of the UAST specification...

Comment (abeaumont) [inline community.md]: I preferred to stay simple (that's something the BIP1 already explains), but maybe not the best wording.

Comment (abeaumont) [inline community.md]: It'd be enough if I replace deeper by design?

**Comment by mcuadros** [comment]
@abeaumont CI failling

```
- /site/community.html
  *  linking to proposals/bip-001.md#workflow, but workflow does not exist (line 391)
     <a href="proposals/bip-001.md#workflow">BIP1</a>
htmlproofer 3.7.2 | Error:  HTML-Proofer found 1 failure!

```

**Comment by abeaumont** [comment]
Thanks, fixed.

**Pull request merged**